torch/serialization.py

```diff
@@ -274,6 +274,17 @@ def add_safe_globals(safe_globals: List[Any]) -> None:
     added to this list can be called during unpickling, classes could be instantiated
     and have state set.
 
+    Each item, ``i`` in the list can either be a function/class or a tuple of the form
+    (function/class, string) where string is a string representing ``{__module__}.{__name__}``
+    of the function/class.
+
+    If a single item ``i`` is passed, ``i`` will be the associated function/class for
```

Comment:
Within the serialized format, each function is identified with it's full path as ``{__module__}.{__name__}``. When calling this API, you can provide this full path that should match the one in the checkpoint otherwise the default `{fn.__module__}.{fn.__name__}` will be used.